tripleo-ciscoaci/puppet/services/ciscoaci-ml2.yaml

```diff
@@ -177,6 +177,10 @@ parameters:
   AciExternalRoutedDomain:
     type: string
     default: ""
+  ACIEnableBondWatchService:
```

Comment:
I'm not as familiar with how this works in puppet, but is ACIEnableBondWatchService defined twice (i.e. once in ciscoaci-ml2.yaml and ciscoaci-compute.yaml), and if so, can/should they be combined into a single variable that's used in both places?